setup.py

```diff
@@ -65,6 +65,7 @@ def run_command(args):
                                ('DTLS_CHECK_CONTENTTYPE', '1'),
                                ('_GNU_SOURCE', '1'),
                                 ('NDEBUG', '1'),
+                                ('uint8_t', 'u_int8_t'),
```

Comment:
this should not be necessary

Reply:
It should not and if you build tinydtls separately, it will build just fine.
I have not worked with cython before and dont understand the build system, but if tinydtls is build from the setup.py it will stop with: 
DTLSSocket/tinydtls/sha2/sha2.c:632:24: error: unknown type name ‘uint8_t’

Reply:
looks like a missing import upstream, this is probably the best we can do